backend/typescript/utilities/emailUtils.ts

```diff
@@ -26,26 +26,38 @@ export const emailFooter = `
   </footer>
   `;
 
-export const cancellationEmail = (mainLine: string, name: string): string => {
+export const cancellationEmail = (
+  mainLine: string,
+  name: string,
+  isVolunteer?: boolean,
+): string => {
   return `
       <html >
       ${emailHeader}
       <body>
         <h2 style="font-weight: 700; font-size: 16px; line-height: 22px; color: #171717;">Hey there ${name}!</h2>
         <p style="font-weight: 400; font-size: 16px; line-height: 24px; color: #171717;">${mainLine}
         </p>
-        <table cellspacing="0" cellpadding="0">
-            <tr>
-              <td align="center" width="255" height="44" bgcolor="#C31887" style="-webkit-border-radius: 6px; -moz-border-radius: 6px; border-radius: 6px; color: #ffffff; display: block;">
-                  <a href="https://communityfridgekw.web.app/" style="font-size:14px; font-weight: bold; font-family:sans-serif; text-decoration: none; line-height:40px; width:100%; display:inline-block">
-                  <span style="color: #FAFCFE;">
-                  View Dashboard
-                  </span>
-                  </a>
-              </td>
-            </tr>
-        </table>
-        <br/>
+        ${
+          isVolunteer
+            ? `<div style=" display:in-line-block;" ><p style="font-weight: 400; font-size: 16px; line-height: 24px; color: #171717; display:in-line-block;">
+        We apologize for the cancellation but would love to stay connected! If you are interested and available for another shift, 
+        <a href="https://communityfridgekw.web.app/volunteer-shifts"  style=" color: #C31887; font-size:16px; font-family:sans-serif; text-decoration: underline; line-height:24px; display:inline-block"> please sign up here. </a>
```

Comment:
the url should be: https://schedule.communityfridgekw.ca. Also where is this copy, I could not find this in the excel sheet

Reply:
<img width="1275" alt="Screen Shot 2022-04-18 at 8 26 47 PM" src="https://user-images.githubusercontent.com/37950626/163897136-321acfb8-5545-45f6-a6a1-641e15ad04c1.png">
 I think it's this one?
@Shehryar21

Reply:
oh yeah gotcha. Also the apologize stuff in emailUtils is added in this pr, so once this gets merged in, I can use that function instead.